packages/frontend/src/presentation/screens/Home/Home.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { View } from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+import { styles } from './styles';
+import { Input } from '../../components/Input';
+
+export const Home: React.FC = () => {
```

Comment:
Num entendi porque a home tem um input.
Isso nao deveria ser uma tela de filtro de busca?

Reply:
Sim, mas a pessoa nao vai poder digitar o nome de uma carta e pesquisar pelo nome nao? igual é feito no mtg familiar e no mtg search?

Reply:
Mas porque isso seria o home? Porque nao dar um nome que melhor descreve a funcionalidade. Esta tela pode ser a home agora, mas depois pode ser outra.

Reply:
Eu to criando um componente para Filtro de busca, acredito que eh um componente que melhor descreve o intuito dele